specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/preview/2020-09-01-preview/databoxedge.json

```diff
@@ -756,12 +756,58 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataBoxEdge/dataBoxEdgeDevices/{deviceName}/generateCertificate": {
+      "post": {
+        "tags": [
+          "Devices"
+        ],
+        "description": "Generates certificate for activation key.",
+        "operationId": "Devices_GenerateCertificate",
+        "parameters": [
+          {
+            "name": "deviceName",
```

Comment:
Suggestion is to put `deviceName` after `resourceGroupName`. But if most API in PR is written this way, no need to change.

Reply:
ok thanks